op-node/rollup/sequencing/sequencer.go

```diff
@@ -488,6 +490,8 @@ func (d *Sequencer) startBuildingBlock() {
 		return
 	}
 
+	recoverMode := d.recoverMode.Load()
```

Comment:
I think the rpc would only set this atomic bool, but not change the l1 origin selector.

If we want to allow to change this via rpc, we either need to also set it in the l1 origin selector, or we do need to thread the bool through to the `FindL1Origin` call... not sure what's better.